test/CircularChartGallery/CircularUIChartGallery.csproj

```diff
@@ -3,7 +3,8 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>tizen40</TargetFramework>
-    <RuntimeIdentifier>tizen-x86</RuntimeIdentifier>
+    <!-- <RuntimeIdentifier>tizen-x86</RuntimeIdentifier> -->
```

Comment:
Is this intended?

Reply:
Yes. `tizen-x86` can be run in the emulator. for running in watch devices ,  I changed from `RuntimeIdentifier` to `CopyLocalRuntimeTargetAssets`. I remain this code for emulator only tpk.